slides/en/index.html

```diff
@@ -298,9 +338,18 @@ <h2>Exercise 1</h2>
             </div>
 
             <aside class="notes">
-              - Demonstrate how to check process history using the administrator view. Speak about the value of graphical monitoring from Enterprise edition.<br/>
-              - Demonstrate how to export the content of the workspace (one diagram / the whole workspace).<br/>
-              - Demonstrate how to import a correction and deal with conflict.
+		      2.00<br /><br />
```

Comment:
@PierrickVouletBonitasoft why did you removed the existing comments? I think having the list of actions to demonstrate is useful.

Reply:
I had rephrased between [] brackets in the same session to highlight it is a demo. The administrator view and graphical monitoring were missing though.